metaxe/templates/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en_US">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>XE Apps - Oregon State University</title>
+    <link rel="Shortcut Icon" href="https://developer.oregonstate.edu/sites/all/themes/doug-fir/favicon.ico" />
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css">
+    <!-- Font Awesome -->
+    <link href="https://netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+    <!-- OSU Style-->
+    <style>
+      {% include 'osu.css' %}
+    </style>
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <header>
+      <a href="http://oregonstate.edu"><img id="logo" src="http://oregonstate.edu/images/logo.jpg?v=1" alt="Oregon State University" width="215" height="78"/></a>
+      <a id="site-title"><h1>XE Apps</h1></a>
+    </header>
+
+    <section class="content">
+      <div class="container">
+        <div class="row">
+          <form class="form">
+            <div class="col-md-4 col-md-offset-5 col-sm-5 col-sm-offset-3">
+              <div class="form-group">
+                <label for="filter-name">Name</label><br>
+                <input id="filter-name" name="q" type="text" class="form-control" value="{{request.args.q}}">
+              </div>
+            </div>
+            <div class="col-sm-2">
+              <div class="form-group">
+                <label for="filter-instance">Instance</label><br>
+                <select id="filter-instance" name="instance" class="form-control">
+                  <option value="" >Any</option>
+                  {% for instance in config.INSTANCES %}
+                    <option value="{{instance}}" {% if request.args.instance == instance %}selected{% endif %}>
+                      {{- instance -}}
+                    </option>
+                  {% endfor %}
+                </select>
+              </div>
+            </div>
+            <div class="col-sm-1">
+              <div class="form-group">
+                <label><br>
+                <button class="btn btn-primary">Filter</button>
```

Comment:
Maybe we should add a "clear" or "reset" button around this area to make the instance set to "Any" and clear the search box.